VibeReads/src/app/modules/auth/signup/signup.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators, AbstractControl } from '@angular/forms';
+
+import { ValidationMessages } from 'app/constants';
+
+import { AuthService , UserService} from '@shared/services/';
+import { strongPasswordValidator ,uniqueUsernameValidator, uniqueEmailValidator, emailValidator } from '@shared/validators/';
```

Comment:
```suggestion
import { strongPasswordValidator ,uniqueUsernameValidator, uniqueEmailValidator, emailValidator } from '@shared/validators';
```

Check at other places also